packages/viewer/src/lib/rendering/matrices.ts

```diff
@@ -155,3 +155,29 @@ export function makeLookAtMatrix(
   ];
   /* eslint-enable prettier/prettier */
 }
+
+export function experimentalMakeTransformationMatrix(
```

Comment:
I will plan on including this in the web-sdk, but I do not think this is the right place - and obviously once we get through the review process, I will plan to remove `experimental` from the name.

Looking for feedback on where to store this? I was thinking in the Matrix4.ts or potentially in the transform-widget `utils.ts` file